classes/sdimg.bbclass

```diff
@@ -120,6 +125,16 @@ EOF
     # Call WIC
     IMAGE_CMD_wic
 
+    # Embed boot loader in boot sector.
```

Comment:
Might want to edit the comments and commit message, technically only the first 512 bytes is the boot sector. instead, we're actually writing u-boot (with FSL's header) at a specific offset so that iMX.6 ROM will find it.